omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,59 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',
+    function ($scope,TagService,ngDialog) {
+
+       /**
+         * Method to load Tags
+         */
+        $scope.loadTags =function (personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
It's because you are really mixing 2 things into one function, you had an init function which should have kept which should take the personUuid argument and all it should do is store it in the scope, load tags should be a separate one with no argument and should reference the personUuid from the scope.